phoenix-core-client/src/main/java/org/apache/phoenix/query/QueryConstants.java

```diff
@@ -708,7 +709,8 @@ enum JoinType {INNER, LEFT_OUTER}
             CLUSTER_ROLE_1 + " VARCHAR," + // Role for peer cluster might not be accurate, we will use only local role for recovery if needed
             CLUSTER_ROLE_2 + " VARCHAR," + // Role for peer cluster might not be accurate, we will use only local role for recovery if needed
             POLICY + " VARCHAR," +  // There should be only one policy for an HA group
-            VERSION + " BIGINT," +  // Version should be incremented for Admin Updates, only for CLUSTER_URLs and REGISTRY_TYPE
+            VERSION_1 + " BIGINT," +  // Version should be incremented for Admin Updates, only for CLUSTER_URLs and REGISTRY_TYPE
```

Comment:
_1 and _2 designate the version of two separate clusters, but that isn't clear. VERSION_1 and VERSION_2 may be two version constants or who knows.  

VERSION_CLUSTER_1 or VERSION_CLUSTER_2 are possible names that make it clear. 

Getting this right matters because there will be compatibility concerns if these are changed later.